iphone/cli/commands/_build.js

```diff
@@ -2916,6 +2920,48 @@ iOSBuilder.prototype.createXcodeProject = function createXcodeProject(next) {
 					comment: xobjs.PBXNativeTarget[targetUuid].productReference_comment
 				});
 
+				// find the extension frameworks and resources group
+				for (var key in extObjs.PBXGroup) {
```

Comment:
Would it make sense to use `Object.keys(extObjs.PBXGroup).forEach(function (key) {} ` here? Just a minor styling thing. 😊